common/src/View/GamesPreferencePane.cpp

```diff
@@ -41,8 +42,11 @@
 #include <QPushButton>
 #include <QStackedWidget>
 #include <QWidget>
+#include <QDesktopServices>
 
 #include "IO/ResourceUtils.h"
+#include "IO/DiskIO.h"
+#include "FileLogger.h"
```

Comment:
This include goes to the top of the list after "Exceptions.h".

Reply:
I assume by "This include" you mean specifically and only FileLogger, and have moved it in the latest commit